src/scenes/HomePage/components/Hero/Hero.module.scss

```diff
@@ -149,35 +149,3 @@
   }
 }
 
-@keyframes gradient-x {
```

Comment:
don't we need this code? 
anything changed after you removed it?

Reply:
Not needed,
nothing changed